helloWorld.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Página</title>
```

Comment:
Cambias "Página" por "Páginas"
```suggestion
    <title>Páginas</title>
```